lib/environment.ts

```diff
@@ -0,0 +1,5 @@
+export function clientInfo(): string {
```

Comment:
This is really small, so I'm wondering if it is worth the trouble keeping this in its own file. Leaving it here for now, for consistency for the other packages.

And speaking of other packages, this code is identical to the equivalent piece in `@percy/nightmare` and in `@percy/cypress`, so wondering if we should start a sort of `@percy/core` package or some other way of re-using this code (and maybe the `_agentJsFilepath` code?). Probably overkill at this point, but something to think about as the packages grow.

Reply:
hmmm I wonder if `@percy/agent` could somehow deliver this path? Agreed a `@percy/core` package is probably overkill.

It is nice to separate the concerns here into a different file and also keep the SDKs consistent. For some SDKs this file is more involved.